ANSWERS.md

```diff
@@ -1,6 +1,17 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 1. Describe the biggest difference between `.forEach` & `.map`.
+While they both iterate over a list, .map will make the needed changes and return a new list. .foreach doesnt actually return anything, it just copies the funtion over each array item and stops with the next line of code. 
+
```

Comment:
That's right! Just be careful to not make typos. It'll help identify you as a more reliable source of information.